### PR TITLE
SSBN to Lycan (for the final time i swear) major revamp to formats

### PR DESCRIPTION
**Pull request by force**

### CODE REVIEW

**Comment by ImLycan** [comment]
tested and works btw

File: mods/wlssb/statuses.js [hunk 1]
```diff
@@ -158,16 +158,18 @@ exports.BattleStatuses = {
...
 		},
 	},
-	ssbn640: {
+	lycaniumz: {
 		exists: true,
-		onStart: function (pokemon) {
-			this.add('c', '%SSBN-640', 'hey, im just here to shitpost or whatever.');
+		onStart: function (source) {
+			this.add('c', '%Lycanium Z', 'Heyyyyyyyyyyyyyyyyyyyyyy');
+			source.types = ["Fighting", "Ghost"];
```
Comment (HoeenCoder) [inline mods/wlssb/statuses.js]: Forcing the foes ability to normalize and making yourself part ghost is too OP.

Comment (ImLycan) [reply mods/wlssb/statuses.js]: Ok, PM'd u on discord

Comment (HoeenCoder) [reply mods/wlssb/statuses.js]: I guess thats ok

**Comment by force** [comment]
eh you know what? Ill just be ssbn-640 over there

**Pull request closed without merging**